src/lib/schematics/update/material/data/constructor-checks.ts

```diff
@@ -26,4 +26,13 @@ export const constructorChecks = [
 
   // https://github.com/angular/material2/pull/9775
   'MatCalendar',
+
+  // TODO(devversion): The constructor check rule doesn't distinct data based on the target
```

Comment:
I think there's a verb missing before "distinct"

Reply:
Good catch. Should be good now.